mission-control-client/src/components/dashboard/products/Product.js

```diff
@@ -5,9 +5,9 @@ const Product = props => {
 
   return (
     <div
-      onClick={() => props.setActiveProduct(props.i)}
+      onClick={() => props.setActiveProduct(props.el)}
       id={
-        props.active === props.i
+        props.active.id && props.active.id === props.i
```

Comment:
For clarification, I think I'd say `props.active.id && (props.active.id === props.i)`.
I get what you're sayin' though.